### ISSUE TITLE
Request update library

### ISSUE THREAD

**Issue by kristiyanP** [airbnb/paris#75]
It's been a while since the last release of the library.
It would be nice if you could release an update with all the fixes that are already merged on master.

Thank you

**Comment by kristiyanP** [comment]
for anyone needing paris updated to  https://github.com/airbnb/paris/commit/0e19c066a2f99ca28c6c9f33459c37e5f9763447
you can get it by linking the following

```gradle
dependencies {
    implementation 'com.bandyer:paris:1.0.0'
    // If you are using Paris annotations
    implementation 'com.bandyer:paris-annotations:1.0.0'
    annotationProcessor 'com.bandyer:paris-processor:1.0.0'
}
```

**Comment by ngsilverman** [comment]
Thanks @kristiyanP! I will be releasing a new version (very) soon! I've been holding off for more testing, we haven't been able to use kapt at Airbnb until a few days ago.

**Comment by ngsilverman** [comment]
Also note you shouldn't need to specify the annotations dependency anymore (you get it transitively from paris module now).

**Comment by kristiyanP** [comment]
> Thanks @kristiyanP! I will be releasing a new version (very) soon! I've been holding off for more testing, we haven't been able to use kapt at Airbnb until a few days ago.

Great!

> Also note you shouldn't need to specify the annotations dependency anymore (you get it transitively from paris module now).

Yes you are totally right!

**Comment by ngsilverman** [comment]
@kristiyanP I just released [v1.1.0](https://github.com/airbnb/paris/releases/tag/v1.1.0) !
